### PR TITLE
Exe api

### PR DESCRIPTION
**Pull request by lailiel**

### CODE REVIEW

**Review by boydstacken — APPROVED** [review]
nice!

File: assets/js/script.js [hunk 1]
```diff
@@ -43,13 +51,16 @@ function generateWorkout() {
...
-  }
-}
+  response.json([0]).then(function (data) {
+    console.log(data[randomEx1])
+     console.log(data[randomEx2])}
+     
+  );
+})}
+};
+let pageBody = $("#page-content");
```
Comment (jayson-baker) [inline assets/js/script.js]: Need to delete lines 61 - 63.

**Review by jayson-baker — CHANGES_REQUESTED** [review]
Need to delete lines 61 - 63

**Review by jayson-baker — APPROVED** [review]
Looks good!

**Pull request merged**